tests/app/va/va_profile/test_va_profile_v3.py

```diff
@@ -0,0 +1,103 @@
+import json
+import pytest
+
+
+from app.va.va_profile import VAProfileClient
+from app.models import RecipientIdentifier
+from app.va.identifier import IdentifierType, transform_to_fhir_format, OIDS
+
+
+MOCK_VA_PROFILE_URL = 'http://mock.vaprofile.va.gov'
+
+
+@pytest.fixture(scope='function')
+def mock_va_profile_client(mocker, notify_api):
+    with notify_api.app_context():
+        mock_logger = mocker.Mock()
+        mock_ssl_key_path = 'some_key.pem'
+        mock_ssl_cert_path = 'some_cert.pem'
+        mock_statsd_client = mocker.Mock()
+
+        client = VAProfileClient()
+        client.init_app(mock_logger, MOCK_VA_PROFILE_URL, mock_ssl_cert_path, mock_ssl_key_path, mock_statsd_client)
+
+        return client
+
+
+@pytest.fixture(scope='module')
+def mock_response():
+    with open('tests/app/va/va_profile/mock_response.json', 'r') as f:
+        return json.load(f)
+
+
+@pytest.fixture(scope='module')
+def recipient_identifier():
+    return RecipientIdentifier(notification_id='123456', id_type=IdentifierType.VA_PROFILE_ID, id_value='1234')
+
+
+@pytest.fixture(scope='module')
+def id_with_aaid(recipient_identifier):
+    return transform_to_fhir_format(recipient_identifier)
+
+
+@pytest.fixture(scope='module')
+def oid(recipient_identifier):
+    return OIDS.get(recipient_identifier.id_type)
+
+
+def test_get_email_from_profile_v3_calls_endpoint_and_returns_email_address(
```

Comment:
According to our [testing guide](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Process/testing_guide.md) all tests should start with one of the following
- test_ut_... (for unit test)
- test_wt_... (for white-box test)
- test_it_... (for integration test)

Reply:
Ah, missed that part.